packages/api/src/controllers/ValidatorsController.js

```diff
@@ -30,7 +30,9 @@ class ValidatorsController {
         isActive,
         validator.proposedBlocksAmount,
         validator.lastProposedBlockHeader,
-        ProTxInfo.fromObject(proTxInfo)))
+        proTxInfo
```

Comment:
I think we should pull up fresh changes from the master

Reply:
✅